RFC/src/RFC-0301_NamespaceRegistration.md

```diff
@@ -0,0 +1,236 @@
+unspent, the RAID tuple will be valid, but when all or part of it is spent, the RAID tuple will disappear from the 
+blockchain. Such a UTXO will thus be "colored" while unspent as it will have different properties to a normal UTXO. It 
+will also be possible to recreate the original RAID tuple by registering it using the original `Hash256(PubKey || FQDN)`.
+
+Thinking about the makeup of the `RAID_ID` it is evident that it can easily be calculated on the fly using the public 
+key and FQDN, both of which values will always be known. The biggest advantage having the RAID tuple on the [base 
+layer] is that of embedded consensus, where it will be validated (as no duplicates can be allowed) and mined before it 
+can be used. However, this comes at the cost of more complex code, a more elaborate asset registration process and 
+higher asset registration fees.
+
+This document explores the creation and use of RAID TXT records to link asset issuer specified domain names with 
+digital assets on the [DAN], without RAID_IDs being registered on the [base layer].
+
+
+
+## Requirements
+
+
+
+### OpenAlias TXT DNS Records
+
+An OpenAlias TXT DNS record [[1]] on a FQDN is a single string and starts with "*oa1:\<name\>*" field followed by a 
+number of key-value pairs. Standard (optional) key-values are: "*recipient_address*"; "*recipient_name*"; 
+"*tx_description*"; "*tx_amount*"; "*tx_payment_id*"; "*address_signature*" and "*checksum*". Additional key-values 
+may also be defined. Only entities with write access to a specific DNS record will be able to create the required TXT 
+DNS record entries.
+
+**Req** - Integration with public DNS records MUST be used to ensure valid ownership of an FQDN that needs to be 
+linked to a [digital asset] on the DAN. 
+
+**Req** - The OpenAlias TXT DNS record MUST be constructed as follows:
+
+| OpenAlias TXT DNS Record Field | OpenAlias TXT DNS Record Data                                |
+| ------------------------------ | ------------------------------------------------------------ |
+| oa1:\<name\>                   | "oa1:tari_raid"                                              |
+| fqdn                           | \<FQDN\>                                                     |
+| public_key                     | \<256 bit public key in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| raid_id                        | \<`RAID_ID` in hexadecimal format excluding the leading "0x" (44 characters)\> |
+| public_nonce                   | \<256 bit public nonce in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| signature                      | \<[Asset issuer]'s 256 bit Schnorr signature (bound to the `RAID_ID`), in hexadecimal format excluding the leading "0x" (64 characters)\> |
+| asset_description              | \<Optional description\>                                     |
```

Comment:
I dont think this is required?

Reply:
It is marked as optional, thus permitted but not required. 
- A typical use could be that Disney creates a range of cartoon character digital assets linked to one RAID_ID, and a loyalty program linked to another RAID_ID.
- Its presence also does not imply we have to accommodate it in the DAN, it can just live in the TXT DNS record.

Reply:
Personally I dont this will happen, as It now requires you to manage extra keys. 

But, I am against this because I dont think we should store extra info here just because we can. From a use case it much more likely that the user already has the asset open, he just wants to confirm the dns. The Dan asset will already have a description. This means the info is duplicate.

Reply:
Remember that we cant search it, we can only search on the DNS. Which means we already have asset id and description. 

Something to remember is that a SPF TXT record can only be 255 char's long.

Reply:
See https://tools.ietf.org/html/rfc7208#section-3.3
Can be a combination of strings, but we should stay within 512 characters.

Reply:
1. Name changed from `asset_description` to `description`

1. TXT DNS record length limitations addressed